### PR TITLE
Add Development scripts with ansible- prefix for now

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

Eventually, I want to rewrite our CI (which currently depends on the up.sh and down.sh committed to the repo), then add the scripts I just shared here instead.  That way the galaxy-operator will be exactly like the other 5 operators.

But for now, I'll just add these scripts with the `ansible-` prefix so they don't conflict.

##### ADDITIONAL INFORMATION

Usage:

```
export QUAY_USER=username
export NAMESPACE=galaxy
export TAG=test
./ansible-up.sh
```

### CODE REVIEW

File: ansible-up.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+# -- Create namespace
+# oc new-project $NAMESPACE
+kubectl create namespace $NAMESPACE
+
+# -- Delete old operator deployment
+kubectl delete deployment galaxy-operator-controller-manager
+
+# -- Create Pull Secret
+kubectl create secret generic redhat-operators-pull-secret --from-file=.dockerconfigjson=/home/chadams/.docker/config.json --type=kubernetes.io/dockerconfigjson
```
Comment (drodowic) [inline ansible-up.sh:86]: `/home/chadams` should be switched to accommodate anyone's home directory, e.g.  `~`

Comment (rooftopcellist) [reply ansible-up.sh]: Thanks, I fixed this by dynamically referecing the home directory:

```
kubectl create secret generic redhat-operators-pull-secret --from-file=.dockerconfigjson=$HOME/.docker/config.json --type=kubernetes.io/dockerconfigjson
```

`~` didn't work btw.  With that, I get this error even though the directory exists.

```
error: error reading ~/.docker/config.json: no such file or directory
```

**Comment by dsavineau** [comment]
@rooftopcellist I think bumping to python 3.12 is maybe too much for the current CI stuff (ie: this would require more work on updating scripts, some dependencies, etc...)
As an intermediate step, we should try using python 3.9

**Comment by dsavineau** [comment]
> @rooftopcellist I think bumping to python 3.12 is maybe too much for the current CI stuff (ie: this would require more work on updating scripts, some dependencies, etc...) As an intermediate step, we should try using python 3.9

Looks like you updated with this just before my suggestion :)

**Review by dsavineau — APPROVED** [review]

**Pull request merged**